clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -243,6 +247,14 @@ void UnwrappedLineParser::parse() {
       Callback.consumeUnwrappedLine(Line);
     }
     Callback.finishRun();
+    TotalLinesProcessed += Lines.size();
+    if ((TotalLinesProcessed / 10000) > LastReport) {
```

Comment:
Do we need this verbose output?

Reply:
My bad, I left that in there by accident. Removed now.